src/schema/root.js

```diff
@@ -91,7 +91,7 @@ type Query {
   ris(pid:String!):String!
   relatedSubjects(q:[String!]!, limit:Int ): [String!] @complexity(value: 3, multipliers: ["q", "limit"])
   inspiration(language: LanguageCode, limit: Int): Inspiration! 
-  orderStatus(orderId: String!): OrderStatusResponse!
+  orderStatus(orderIds: [String]!): [OrderStatusResponse]!
```

Comment:
orderIds input må gerne være [String!]! istedet for bare [String]! - så sikre vi at input i listen rent faktisk er strings